network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CalculatedBus.java

```diff
@@ -147,6 +148,40 @@ public Bus setAngle(double angle) {
         return this;
     }
 
+    @Override
+    public double getFictitiousP0() {
+        return voltageLevelResource.getAttributes().getNodeToFictitiousP0().values().stream().reduce(0.0, Double::sum);
+    }
+
+    @Override
+    public Bus setFictitiousP0(double p0) {
+        if (voltageLevelResource.getAttributes().getNodeToFictitiousP0() != null) {
+            voltageLevelResource.getAttributes().getNodeToFictitiousP0().replaceAll((k, v) -> 0.0);
+        }
+        getVoltageLevel().getNodeBreakerView().setFictitiousP0(voltageLevelResource.getAttributes().getNodeToFictitiousP0().keySet().stream()
```

Comment:
voltageLevelResource.getAttributes().getNodeToFictitiousP0()can be null, I think ...